`make clusters` fails

<!-- Please use this template while reporting a bug and provide as much info as
possible. Not doing so may result in your bug not being addressed in a timely
manner. Thanks!

If the matter is security related, please disclose it privately to the
Submariner Owners: https://github.com/orgs/submariner-io/teams/submariner-core
-->


**What happened**:  Getting errors when running `make clusters` or `make deploy` as part of the getting started guide for kind. 

**What you expected to happen**:
Kind clusters should be created.

**How to reproduce it (as minimally and precisely as possible)**:
Tried both on CentOS 7.8 and macOS  11.6

**Anything else we need to know?**:

kubectl, subctl, docker, kind already installed properly.

CentOS output: 

```
$ make clusters
Makefile:126: Makefile.dapper: No such file or directory
Downloading Makefile.dapper
Downloading dapper
Submariner Dapper
Downloading Dockerfile.dapper
docker network create -d bridge kind
Error response from daemon: network with name kind already exists
make: [clusters] Error 1 (ignored)
./.dapper   -- make --debug=b clusters
Sending build context to Docker daemon   9.09MB
Step 1/8 : ARG BASE_BRANCH
Step 2/8 : FROM quay.io/submariner/shipyard-dapper-base:${BASE_BRANCH}
 ---> 1a2513f0257b
Step 3/8 : ARG PROJECT
 ---> Using cache
 ---> b14ff42e2fb8
Step 4/8 : ENV DAPPER_ENV="QUAY_USERNAME QUAY_PASSWORD CLUSTERS_ARGS DEPLOY_ARGS CLEANUP_ARGS E2E_ARGS RELEASE_ARGS MAKEFLAGS FOCUS SKIP PLUGIN E2E_TESTDIR GITHUB_USER GITHUB_TOKEN"     DAPPER_SOURCE=/go/src/github.com/submariner-io/${PROJECT} DAPPER_DOCKER_SOCKET=true
 ---> Using cache
 ---> 75002b8b7e4d
Step 5/8 : ENV DAPPER_OUTPUT=${DAPPER_SOURCE}/output
 ---> Using cache
 ---> 9a74234220de
Step 6/8 : WORKDIR ${DAPPER_SOURCE}
 ---> Using cache
 ---> 48d3b8677841
Step 7/8 : ENTRYPOINT ["/opt/shipyard/scripts/entry"]
 ---> Using cache
 ---> 7c3f8ac6cc15
Step 8/8 : CMD ["sh"]
 ---> Using cache
 ---> 88b117083c88
Successfully built 88b117083c88
Successfully tagged submariner:devel
[submariner]$ trap chown -R 1001:1001 . exit
[submariner]$ mkdir -p bin dist output
[submariner]$ make --debug=b clusters
GNU Make 4.3
Built for x86_64-redhat-linux-gnu
Copyright (C) 1988-2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Reading makefiles...
make: cmp: Operation not permitted
make: /bin/bash: Operation not permitted
make: /bin/bash: Operation not permitted
make: /bin/bash: Operation not permitted
make: /bin/bash: Operation not permitted
make: /bin/bash: Operation not permitted
make: /bin/bash: Operation not permitted
make: /bin/bash: Operation not permitted
Updating makefiles....
Updating goal targets....
 File 'clusters' does not exist.
Must remake target 'clusters'.
/opt/shipyard/scripts/clusters.sh --settings /go/src/github.com/submariner-io/submariner/.shipyard.e2e.yml
make: /bin/bash: Operation not permitted
make: *** [/opt/shipyard/Makefile.inc:106: clusters] Error 127
[submariner]$ make --debug=b clusters
make: *** [clusters] Error 2
```

Mac output: 
```
% make clusters
Makefile:126: Makefile.dapper: No such file or directory
Downloading Makefile.dapper
Downloading dapper
Submariner Dapper
Downloading Dockerfile.dapper
docker network create -d bridge kind
Error response from daemon: network with name kind already exists
make: [clusters] Error 1 (ignored)
./.dapper   -- make --debug=b clusters
[+] Building 10.2s (3/3) FINISHED
 => [internal] load build definition from Dockerfile.dapper                                                                                                                                        0.1s
 => => transferring dockerfile: 524B                                                                                                                                                               0.0s
 => [internal] load .dockerignore                                                                                                                                                                  0.0s
 => => transferring context: 2B                                                                                                                                                                    0.0s
 => ERROR [internal] load metadata for quay.io/submariner/shipyard-dapper-base:devel                                                                                                              10.0s
------
 > [internal] load metadata for quay.io/submariner/shipyard-dapper-base:devel:
------
failed to solve with frontend dockerfile.v0: failed to create LLB definition: failed to do request: Head https://quay.io/v2/submariner/shipyard-dapper-base/manifests/devel: Bad Gateway
sed: 1: "/"DAPPER_CP=/{s/[[:spac ...": extra characters at the end of q command
sed: 1: "/"DAPPER_ENV=/{s/[[:spa ...": extra characters at the end of q command
sed: 1: "/"DAPPER_SOURCE=/{s/[[: ...": extra characters at the end of q command
sed: 1: "/"DAPPER_DOCKER_SOCKET= ...": extra characters at the end of q command
sed: 1: "/"DAPPER_RUN_ARGS=/{s/[ ...": extra characters at the end of q command
docker: invalid reference format: repository name must be lowercase.
See 'docker run --help'.
make: *** [clusters] Error 125
```


**Environment**:
- Diagnose information (use `subctl diagnose all`):
- Gather information (use `subctl gather`):
- Cloud provider or hardware configuration:
- Install tools:
- Others:

Would be helpful also to have docs on what is the required configuration for kind clusters when running kind CLI manually not using make file?  (CIDRs, CNI, nodes, etc..) 
Tried also to setup kind clusters manually with non-overlapping CIDRs but gateways on both clusters cannot connect.
Pardon if this bug report is due to a newcomer's mistake.

Tried also Ubunto 18.04, same failure as CentOS:

```
% make clusters
Makefile:126: Makefile.dapper: No such file or directory
Downloading Makefile.dapper
Downloading dapper
Submariner Dapper
Downloading Dockerfile.dapper
docker network create -d bridge kind
af765c07730d2d737ab9cef01ff01dd8a7e6d919279487bff4a5f21e362536d9
./.dapper   -- make --debug=b clusters
Sending build context to Docker daemon  9.104MB
Step 1/8 : ARG BASE_BRANCH
Step 2/8 : FROM quay.io/submariner/shipyard-dapper-base:${BASE_BRANCH}
devel: Pulling from submariner/shipyard-dapper-base
fc811dadee24: Pull complete
865443303fbd: Pull complete
7e3d5872c873: Pull complete
8754e7e7fd33: Pull complete
730a0c2def2c: Pull complete
8a98e40c8eb2: Pull complete
f176b9b8e444: Pull complete
b4f15b33efc7: Pull complete
Digest: sha256:befdf30bdcdb0760e5fa54582471b800aab697606c811f898b88d2216edf13f8
Status: Downloaded newer image for quay.io/submariner/shipyard-dapper-base:devel
 ---> 4da2955b6c34
Step 3/8 : ARG PROJECT
 ---> Running in 76c54c18300d
Removing intermediate container 76c54c18300d
 ---> 26ef4e8714a0
Step 4/8 : ENV DAPPER_ENV="QUAY_USERNAME QUAY_PASSWORD CLUSTERS_ARGS DEPLOY_ARGS CLEANUP_ARGS E2E_ARGS RELEASE_ARGS MAKEFLAGS FOCUS SKIP PLUGIN E2E_TESTDIR GITHUB_USER GITHUB_TOKEN"     DAPPER_SOURCE=/go/src/github.com/submariner-io/${PROJECT} DAPPER_DOCKER_SOCKET=true
 ---> Running in 9f7ca6868a90
Removing intermediate container 9f7ca6868a90
 ---> 6e31c8b5c167
Step 5/8 : ENV DAPPER_OUTPUT=${DAPPER_SOURCE}/output
 ---> Running in 18172c2a4e8b
Removing intermediate container 18172c2a4e8b
 ---> 5ae4c03b5f52
Step 6/8 : WORKDIR ${DAPPER_SOURCE}
 ---> Running in ad3a16e53c8a
Removing intermediate container ad3a16e53c8a
 ---> eb0606753ddb
Step 7/8 : ENTRYPOINT ["/opt/shipyard/scripts/entry"]
 ---> Running in f38135a19641
Removing intermediate container f38135a19641
 ---> 07f03d5b8016
Step 8/8 : CMD ["sh"]
 ---> Running in a0f6aa1ef0d2
Removing intermediate container a0f6aa1ef0d2
 ---> aa3b7e90023a
Successfully built aa3b7e90023a
Successfully tagged submariner:devel
[submariner]$ trap chown -R 217779353:10002 . exit
[submariner]$ mkdir -p bin dist output
[submariner]$ make --debug=b clusters
GNU Make 4.3
Built for x86_64-redhat-linux-gnu
Copyright (C) 1988-2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Reading makefiles...
make: cmp: Operation not permitted
make: /bin/bash: Operation not permitted
make: /bin/bash: Operation not permitted
make: /bin/bash: Operation not permitted
make: /bin/bash: Operation not permitted
make: /bin/bash: Operation not permitted
make: /bin/bash: Operation not permitted
make: /bin/bash: Operation not permitted
Updating makefiles....
Updating goal targets....
 File 'clusters' does not exist.
Must remake target 'clusters'.
/opt/shipyard/scripts/clusters.sh --settings /go/src/github.com/submariner-io/submariner/.shipyard.e2e.yml
make: /bin/bash: Operation not permitted
make: *** [/opt/shipyard/Makefile.inc:106: clusters] Error 127
[submariner]$ make --debug=b clusters
Makefile.dapper:24: recipe for target 'clusters' failed
make: *** [clusters] Error 2
```

--- a/.dapper
+++ b/.dapper
@@ -85,4 +85,8 @@ fi
 
 [ -t 1 ] && dockerargs="${dockerargs} -t"
 
-docker run -i --rm $(printf -- " -e %s" $DAPPER_ENV) -e "DAPPER_UID=$(id -u)" -e "DAPPER_GID=$(id -g)" -v "${DAPPER_CP}:${DAPPER_SOURCE}${suffix}" ${dockerargs} ${DAPPER_RUN_ARGS} "${container}" "$@"
+# seccomp is unconfined to avoid problems with clone3 in Fedora 35
+# See https://pascalroeleven.nl/2021/09/09/ubuntu-21-10-and-fedora-35-in-docker/
+# Remove "--security-opt seccomp=unconfined" once all supported container runtimes
+# handle clone3
+docker run -i --rm --security-opt seccomp=unconfined $(printf -- " -e %s" $DAPPER_ENV) -e "DAPPER_UID=$(id -u)" -e "DAPPER_GID=$(id -g)" -v "${DAPPER_CP}:${DAPPER_SOURCE}${suffix}" ${dockerargs} ${DAPPER_RUN_ARGS} "${container}" "$@"

--- a/package/Dockerfile.shipyard-dapper-base
+++ b/package/Dockerfile.shipyard-dapper-base
@@ -55,7 +55,7 @@ RUN dnf -y install --nodocs --setopt=install_weak_deps=False \
 ENV LINT_VERSION=v1.43.0 \
     HELM_VERSION=v3.7.1 \
     KIND_VERSION=v0.11.1 \
-    BUILDX_VERSION=v0.7.0 \
+    BUILDX_VERSION=v0.7.1 \
     GH_VERSION=2.2.0 \
     YQ_VERSION=4.14.1
 

